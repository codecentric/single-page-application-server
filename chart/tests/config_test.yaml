suite: test config
templates:
  - config.yaml
tests:
  - it: should render minimal values
    values:
      - ./values/minimal_values.yaml
    asserts:
      - matchSnapshot: {}
