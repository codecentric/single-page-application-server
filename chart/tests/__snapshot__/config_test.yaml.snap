should render minimal values:
  1: |
    apiVersion: v1
    data:
      config.yaml: |
        default:
          base_href: /
          http:
            enabled: true
            port: 8080
          https:
            enabled: false
            port: 8443
            ssl_certificate: /var/run/secrets/tls/tls.crt
            ssl_certificate_key: /var/run/secrets/tls/tls.key
          spa_config:
            endpoints: {}
    kind: ConfigMap
    metadata:
      labels:
        app.kubernetes.io/component: frontend
        app.kubernetes.io/instance: default
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: RELEASE-NAME
        app.kubernetes.io/version: latest
        helm.sh/chart: single-page-application-server-chart-0.0.0
      name: RELEASE-NAME
